tests/integration/api/test_v1_root.py

```diff
@@ -0,0 +1,99 @@
+#  Copyright 2024 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import pytest
+
+from tests.integration.constants import api_url_v1
+
+
+@pytest.mark.parametrize(
+    "expected_slugs,use_shared_resource",
+    [
+        pytest.param(
+            [
+                "/status/",
+                "/openapi.json",
+                "/openapi.yaml",
+                "/docs",
+                "/redoc",
+                "/auth/session/login/",
+                "/auth/session/logout/",
+                "/auth/token/refresh/",
+                "/users/me/",
+                "/projects/",
+                "/rulebooks/",
+                "/activations/",
+                "/activation-instances/",
+                "/audit-rules/",
+                "/users/",
+                "/event-streams/",
+                "/users/me/awx-tokens/",
+                "/credential-types/",
+                "/eda-credentials/",
+                "/decision-environments/",
+                "/organizations/",
+                "/teams/",
+            ],
+            True,
+            id="with_shared_resource",
```

Comment:
Is there a purpose to `id`?  There's not a third entry in the parameter list.

Reply:
@jshimkus-rh 
`id` is a parameter for pytest. It's the name of the test case. 
```
tests/integration/api/test_v1_root.py::test_v1_root[with_shared_resource] PASSED [ 45%]
tests/integration/api/test_v1_root.py::test_v1_root[no_shared_resource] PASSED [ 45%]
```

Reply:
Learn something new everyday.  I guess I never looked that close at the output from stuff I parameterized.